src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1226,13 +1227,38 @@ public TeleportCause get(final PlayerTeleportEvent e) {
 				return e.getCause();
 			}
 		}, 0);
-		EventValues.registerEventValue(PlayerTeleportEvent.class, Location.class, new Getter<Location, PlayerTeleportEvent>() {
+		//PlayerMoveEvent
+		EventValues.registerEventValue(PlayerMoveEvent.class, Location.class, new Getter<Location, PlayerMoveEvent>() {
 			@Override
 			@Nullable
-			public Location get(final PlayerTeleportEvent e) {
+			public Location get(PlayerMoveEvent e) {
 				return e.getFrom();
 			}
 		}, -1);
```

Comment:
Can you use the time constants from EventValues here?

Reply:
They haven't been merged aren't they?

Reply:
They have: https://github.com/SkriptLang/Skript/blob/ceaa32d7ed6595d723c8bfe7949ed0de2c2553c0/src/main/java/ch/njol/skript/registrations/EventValues.java#L104-L117

Reply:
I think pickled means to use the field reference rather than the actual value of the field.